AnswerKingAPI.yml

```diff
@@ -505,6 +583,18 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemDetails'
+        '404':
+          description: Category with the given id does not exist
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+        '410':
+          description: Category with the given id is retired.
```

Comment:
Think this is also `Category or Tag with the given id is retired`

Reply:
changed